lib/src/utils/feature_flags_manager.dart

```diff
@@ -0,0 +1,90 @@
+import 'package:instabug_flutter/src/generated/instabug.api.g.dart';
+import 'package:instabug_flutter/src/utils/ibg_build_info.dart';
+import 'package:meta/meta.dart';
+
+typedef OnW3CFeatureFlagChange = void Function(
+  bool isW3ExternalTraceIDEnabled,
+  bool isW3ExternalGeneratedHeaderEnabled,
+  bool isW3CaughtHeaderEnabled,
+);
+
+class FeatureFlagsManager implements FeatureFlagsFlutterApi {
+  static InstabugHostApi _host = InstabugHostApi();
+  static FeatureFlagsManager _instance = FeatureFlagsManager();
+
+  /// @nodoc
+  @visibleForTesting
+  // ignore: use_setters_to_change_properties
+  static void $setHostApi(InstabugHostApi host) {
+    _host = host;
+  }
+
+  // FeatureFlagsManager({InstabugHostApi? instabugHostApi}) {
+  //   if (instabugHostApi != null) {
+  //     _host = instabugHostApi;
+  //   }
+  // }
+
+  @visibleForTesting
+  // ignore: use_setters_to_change_properties
+  static void $setFeatureFlagsManager(FeatureFlagsManager featureFlagsManager) {
+    _instance = featureFlagsManager;
+  }
+
+  static bool _isAndroidW3ExternalTraceID = false;
+  static bool _isAndroidW3ExternalGeneratedHeader = false;
+  static bool _isAndroidW3CaughtHeader = false;
+
+  static Future<bool> get isW3ExternalTraceID async {
```

Comment:
Let's use the actual name and change W3* members to W3C*, for example, changing `isW3ExternalTraceID` to `isW3CExternalTraceID`, and for all of the file content as well.